src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -1062,6 +1064,52 @@ private bool ImportIntrinsicCall(MethodDesc method)
 
             switch (method.Name)
             {
+                case "InitializeArray":
+                    if (metadataType.Namespace == "System.Runtime.CompilerServices" && metadataType.Name == "RuntimeHelpers")
+                    {
+                        var fieldSlot = (LdTokenEntry<FieldDesc>)_stack.Pop();
+                        var arraySlot = _stack.Pop();
+
+                        var fieldDesc = fieldSlot.LdToken;
+                        var dataBlob = _compilation.GetFieldRvaData(fieldDesc).GetData(_compilation.NodeFactory, false);
+                        Debug.Assert(dataBlob.Relocs.Length == 0);
+                        var memBlock = dataBlob.Data;
+
+                        // TODO: Need to do more for arches with different endianness?
+                        LLVMValueRef globalDataHolder = LLVM.AddGlobal(Module, LLVM.ArrayType(LLVM.Int8Type(), (uint)memBlock.Length), $"array.init.{_arrayInitializerHolderCount++}");
+                        var llvmInitValues = new LLVMValueRef[memBlock.Length];
+                        for (int i = 0; i < memBlock.Length; ++i)
+                            llvmInitValues[i] = BuildConstInt8(memBlock[i]);
+                        LLVM.SetInitializer(globalDataHolder, LLVM.ConstArray(LLVM.Int8Type(), llvmInitValues));
+                        LLVM.SetGlobalConstant(globalDataHolder, LLVMMisc.True);
+                        LLVM.SetUnnamedAddr(globalDataHolder, LLVMMisc.True);
+                        LLVM.SetLinkage(globalDataHolder, LLVMLinkage.LLVMPrivateLinkage);
+
+                        var argsType = new LLVMTypeRef[]
+                        {
+                            LLVM.PointerType(LLVM.Int8Type(), 0),
+                            LLVM.PointerType(LLVM.Int8Type(), 0),
+                            LLVM.Int32Type(),
+                            LLVM.Int32Type(),
+                            LLVM.Int1Type()
+                        };
+                        MemcpyI8I8I32Function = GetOrCreateLLVMFunction("llvm.memcpy.p0i8.p0i8.i32", LLVM.FunctionType(LLVM.VoidType(), argsType, false));
```

Comment:
It looks like CI is failing with this variable not being declared

Reply:
Right. I forgot to commit corresponding modifications. Will fix soon (not now since that will bother the CI while further updates are still on the way).